### PR TITLE
Update building instruction based on removed code

### PR DESCRIPTION
**Pull request by ptitdoc**
Use removed code from the archlinux installation guide.

### CODE REVIEW

File: building/building-archlinux-template.md [hunk 1]
```diff
@@ -8,77 +8,355 @@ redirect_from:
 - /wiki/BuildingArchlinuxTemplate/
 ---
 
-Template building
-=================
+## Archlinux template building instructions ##
 
-The archlinux VM is now almost working as a NetVM. Based on qubes-builder code, you could find below how to build it and problem that could arise from template building to using archlinux as a netvm:
+<br>
+**These are the instructions for Qubes 3.2. They will take you step by step through the entire process start to finish**
```
Comment (marmarek) [inline building/building-archlinux-template.md]: Shouldn't this be changed to 3.1?
There are much more occurrences of "3.2" on this page. Maybe some general comment here would be useful? Not sure if the instruction as-is will work on 3.1 - probably yes (after `s/3.2/3.1/`).

File: building/building-archlinux-template.md [hunk 2]
```diff
@@ -8,77 +8,355 @@ redirect_from:
...
 
-Download qubes-builder git code
--------------------------------
+*Note: Currently there are no binary packages and it must be compiled from source using the instructions below.*
```
Comment (marmarek) [inline building/building-archlinux-template.md]: And this also update to state it's about 3.1?

**Comment by andrewdavidwong** [comment]
@ptitdoc, just FYI (no rush): I'm waiting for you to respond to @marmarek's questions before merging this.

**Comment by ptitdoc** [comment]
Thanks, I forgot this was not merged and I even created a new pull request. I have yet to find the github function to add a commit to an existing pull request.

File: building/building-archlinux-template.md (hunk 1 above)
Comment (ptitdoc) [inline building/building-archlinux-template.md]: Yes all occurences can be replaced to 3.2.

**Comment by andrewdavidwong** [comment]
@ptitdoc: Have you tried simply pushing commits to the same branch?

**Comment by ptitdoc** [comment]
I added notes related to qubes 3.1. I hope it answers @marmarek questions.

**Comment by marmarek** [comment]
I think it's good enough now. When/if someone will actually test it on 3.1, we can update that note one more time.

**Comment by ptitdoc** [comment]
I have done some cleanup in the markup because I was not able to read the source code anymore :-)

**Pull request merged**